lib/routes/account.js

```diff
@@ -66,7 +66,8 @@ module.exports = function (
               name: isA.string().max(255).required(),
               type: isA.string().max(16).required(),
               pushCallback: isA.string().uri({ scheme: 'https' }).max(255).optional().allow(''),
-              pushPublicKey: isA.string().length(64).regex(HEX_STRING).optional().allow('')
+              // We're not yet ready to store pubkey values, don't let clients submit them.
```

Comment:
@rfk Sorry I'm not understanding the comment here: ` don't let clients submit them.` sounds like the request should be rejected if they key is there? (While the rule is set to `optional()` in hapi?)